src/nni_manager/training_service/kubeflow/kubeflowTrainingService.ts

```diff
@@ -245,7 +245,11 @@ class KubeflowTrainingService implements TrainingService {
         }
 
         trialJobDetail.endTime = Date.now();
-        trialJobDetail.status = 'USER_CANCELED';        
+        if (byAssessor) {
+            trialJobDetail.status = 'EARLY_STOPPED';
+        } else {
+            trialJobDetail.status = 'USER_CANCELED';
+        }
```

Comment:
Codes between line 248-252 are duplicated somewhere. How about wrap it into a util method?

Reply:
good point, thanks